tls/s2n_security_policies.c

```diff
@@ -36,6 +36,20 @@ const struct s2n_security_policy security_policy_default_tls13 = {
     .ecc_preferences = &s2n_ecc_preferences_20200310,
 };
 
+/*
+ * The following security policy is derived from the following specification:
+ * https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-52r2.pdf
+ *
+ * Only supports TLS1.2
+ */
+const struct s2n_security_policy security_policy_default_fips = {
+    .minimum_protocol_version = S2N_TLS10,
```

Comment:
The comment says "Only supports TLS 1.2" but this line implies that it works down to TLS 1.0. What am I missing? Shouldn't it be `S2N_TLS12`?

Reply:
updated comment to specify 1.1 and 1.0. There is an open item if we want to support those version also. see comment https://github.com/aws/s2n-tls/pull/3357#discussion_r894874529